tracex_project/db_results/templates/metrics_dashboard.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Metrics Dashboard</title>
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static '/tracex/css/style.css' %}">
+    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+    <script src="{% static '/db_results/js/metrics-dashboard.js' %}" defer></script>
+</head>
+<body class="main_body">
+<h1>Metrics Dashboard ({{ patient_journey_name }})</h1>
+<div class="dashboard">
+    <h2>General Information</h2>
+    <h3>Database Information</h3>
+    <div class="row">
+        <div class="metric-tile">
+            <h3>Total Patient Journeys</h3>
+            <div class="number-metric">{{ total_patient_journeys }}</div>
+        </div>
+        <div class="metric-tile">
+            <h3>Total Traces in DB</h3>
+            <div class="number-metric">{{ total_traces }}</div>
+        </div>
+    </div>
+    <h3>Trace Related Information</h3>
+    <div class="row">
+        <div class="metric-tile">
+            <h3>Total Activties</h3>
+            <div class="number-metric">{{ total_activities }}</div>
+        </div>
+        <div class="metric-tile">
+            <h3>Total Traces of Current Patient Journey</h3>
+            <div class="number-metric">{{ traces_count }}</div>
+        </div>
+    </div>
+    <h2>Activity Relevance</h2>
+    <div class="row">
+        <div class="metric-tile">
+            <h3>Most Frequent Category</h3>
+            <div class="number-metric">{{ most_frequent_category_count }}</div>
+            <div class="metric-content" id="most_frequent_category">{{ most_frequent_category }} </div>
+            <span class="info-icon">&#9432;</span>
+            <div class="info-text">This metric shows the most frequent relevance category assigned to the extracted activities. Based on the most frequent relevance category the colour of the metric tile changes.</div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="metric-tile">
+            <h3>Activity Relevance Pie Chart</h3>
+            <div class="chart-content">{{ activity_relevance_pie_chart|safe }}</div>
+            <span class="info-icon ">&#9432;</span>
+            <div class="info-text">The pie chart visualizes the distribution of relevance categories for the extracted activities.</div>
+        </div>
+        <div class="metric-tile">
+            <h3>Activity Relevance Bar Chart</h3>
+            <div class="chart-content">{{ activity_relevance_bar_chart|safe }}</div>
+            <span class="info-icon">&#9432;</span>
+            <div class="info-text">The bar chart displays the count of extracted activities in each relevance category.</div>
+        </div>
+    </div>
+    <div class="row">
+        <div class="table-container"> {{ relevance_df|safe}}</div>
+    </div>
```

Comment:
what exactly should be centered?

Reply:
![image](https://github.com/bptlab/TracEX/assets/73845255/7af8ed2e-74b2-44a6-a6a6-5d658c606d91)

since everything is centered with the actual style.css, I will resolve this thread. Please reopen it if there is still an issue